packages/splitter/src/useSplitter.ts

```diff
@@ -101,54 +121,82 @@ export function useSplitter({
   defaultValueNow = min,
   valueNow,
   onChange = () => undefined,
-  rtl = false,
+  placement,
   ...props
 }: IUseSplitterProps): IUseSplitterReturnValue {
+  const primaryPaneId = useId();
```

Comment:
Would it be good to offer the option for the consumer to set some or all of the ID as Garden does in other containers? Recommend grep/ack for "idPrefix" usage across `react-containers`. ymmv

Reply:
Still working on this one.

Reply:
Used Accordion's implementation of `idPrefix` in https://github.com/zendeskgarden/react-containers/pull/405/commits/89d12bd98d65d77fd96956869881b5a0c55691f0